install_config/configuring_sdn.adoc

```diff
@@ -288,18 +287,21 @@ service neutron-plugin-openvswitch-agent restart
 service neutron-vpn-agent restart
 service neutron-server  restart
 ----
-
-When creating the OpenStack instances, disable both port-security and security
-groups in the ports where the container network Flannel interface will be:
-
++
+When creating the {product-title} instances, disable both port security and security
+groups in the ports where the container network flannel interface will be:
```

Comment:
flannel vs Flannel

Reply:
In the flannel docs, they use `flannel` with lower case `f` except at the beginning of sentences where they use `Flannel`. I tried to match that.

Reply:
Line 260 should be lower case then.